test/sender_and_receiver_integration_test.exs

```diff
@@ -114,6 +114,34 @@ defmodule SenderAndReceiverIntegrationTest do
     end
   end
 
+  describe "timeout behaviour" do
+    test "does not reconnect when a receive timeout occurs" do
+      port = 8153
+
+      {:ok, %{pid: _receiver_pid}} =
+        MLLP.Receiver.start(
+          port,
+          SenderAndReceiverIntegrationTest.TestSlowDispatcher
+        )
+
+      {:ok, sender_pid} = MLLP.Sender.start_link({127, 0, 0, 1}, port)
+
+      message =
+        (HL7.Examples.wikipedia_sample_hl7() <>
+           "ZNK|JUNK|" <> String.pad_leading("\r", 100, "Z"))
+        |> HL7.Message.new()
+        |> to_string()
+
+      {:error, _} = MLLP.Sender.send_raw_and_receive_reply(sender_pid, message, 100)
+
+      assert Process.alive?(sender_pid)
+
+      Process.sleep(1000)
```

Comment:
I tested this after removing process.sleep and it seems to work fine every time I test it.

Reply:
That's because I have a `Process.sleep/1` call in the dispatcher,  I'll remove the call here.